### PR TITLE
Add TLS configuration options to Chronograf TLS guides

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#282

This updates the [Configure TLS for Chronograf](https://docs.influxdata.com/chronograf/v1.9/administration/managing-security/#configure-tls-for-chronograf) sections in both Chronograf 1.8 and 1.9 to include more TLS configuration options examples. It also corrects the Docker image used in these examples.

- [x] Rebased/mergeable

### CODE REVIEW

**Review by btasker — APPROVED** [review]

File: content/chronograf/v1.8/administration/managing-security.md [hunk 1]
```diff
@@ -531,29 +531,56 @@ All Chronograf command line options have corresponding environment variables.
...
+1. Specify the certificate file using the `TLS_CERTIFICATE` environment variable or the `--cert` CLI option.
+2. Specify the key file using the `TLS_PRIVATE_KEY` environment variable or `--key` CLI option.
 
-{{% note %}}
+    {{% note %}}
 If both the TLS certificate and key are in the same file, specify them using the `TLS_CERTIFICATE` environment variable (or the `--cert` CLI option).
-{{% /note %}}
+    {{% /note %}}
+
+3. _(Optional)_ Specify which TLS cipher suites to allow using the `TLS_CIPHERS` or `--tls-ciphers` CLI option.
```
Comment (jstirnaman) [inline content/chronograf/v1.8/administration/managing-security.md:541]:
```suggestion
3. _(Optional)_ To specify which TLS cipher suites to allow, use the `TLS_CIPHERS` environment variable or the `--tls-ciphers` CLI option.
```

File: content/chronograf/v1.8/administration/managing-security.md [hunk 2]
```diff
@@ -531,29 +531,56 @@ All Chronograf command line options have corresponding environment variables.
...
+    Chronograf supports all cipher suites in the
+    [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants)
+    and, by default, allows all of them.
+4. _(Optional)_ Specify the minimum and maximum TLS versions to allow with the
+    `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
+    `--tls-min-version` and `--tls-max-version` CLI options.
+    By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
+    unlimited.
```
Comment (jstirnaman) [inline content/chronograf/v1.8/administration/managing-security.md:549]:
```suggestion
4. _(Optional)_ To specify the minimum and maximum TLS versions to allow, use the
    `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
    `--tls-min-version` and `--tls-max-version` CLI options.
    By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
    unlimited.
```

File: content/chronograf/v1.8/administration/managing-security.md [hunk 3]
```diff
@@ -531,29 +531,56 @@ All Chronograf command line options have corresponding environment variables.
...
+  -e TLS_PRIVATE_KEY=/certs/my.key \
+  -e TLS_CIPHERS=TLS_RSA_WITH_AES_256_CBC_SHA,TLS_AES_128_GCM_SHA256 \
+  -e TLS_MIN_VERSION=tls1.2 \
+  -e TLS_MAX_VERSION=tls1.3 \
+  chronograf:{{< current-version >}}
 ```
 
-### Testing with self-signed certificates
+### Test with self-signed certificates
 In a production environment you should not use self-signed certificates, but for testing it is fast to create your own certificates.
```
Comment (jstirnaman) [inline content/chronograf/v1.8/administration/managing-security.md:584]:
```suggestion
To test your setup, you can use a self-signed certificate.

{{% warn %}}
Don't use self-signed certificates in production environments.
{{% /warn %}}

```

File: content/chronograf/v1.7/administration/managing-security.md [hunk 4]
```diff
@@ -482,7 +482,7 @@ TLS_CERTIFICATE=my.crt TLS_PRIVATE_KEY=my.key chronograf
 
 #### Docker example with environment variables
 ```sh
-docker run -v /host/path/to/certs:/certs -e TLS_CERTIFICATE=/certs/my.crt -e TLS_PRIVATE_KEY=/certs/my.key quay.io/influxdb/chronograf:latest
+docker run -v /host/path/to/certs:/certs -e TLS_CERTIFICATE=/certs/my.crt -e TLS_PRIVATE_KEY=/certs/my.key chronograf:{{< current-version }}
```
Comment (sanderson) [inline content/chronograf/v1.7/administration/managing-security.md:485]:
```suggestion
docker run -v /host/path/to/certs:/certs -e TLS_CERTIFICATE=/certs/my.crt -e TLS_PRIVATE_KEY=/certs/my.key chronograf:{{< current-version >}}
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 5]
```diff
@@ -543,7 +544,7 @@ When configured, users can use HTTPS to securely communicate with your Chronogra
 Using HTTPS helps guard against nefarious agents sniffing the JWT and using it to spoof a valid user against the Chronograf server.
```
Comment (jstirnaman) [inline content/chronograf/v1.9/administration/managing-security.md:544]:
```suggestion
HTTPS helps prevent nefarious agents stealing the JWT and using it to spoof a valid user against the server.
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 6]
```diff
@@ -553,29 +554,56 @@ All Chronograf command line options have corresponding environment variables.
 
 **To configure Chronograf to support TLS:**
```
Comment (jstirnaman) [inline content/chronograf/v1.9/administration/managing-security.md:555]:
```suggestion
To configure Chronograf to support TLS, do the following:
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 7]
```diff
@@ -553,29 +554,56 @@ All Chronograf command line options have corresponding environment variables.
...
+1. Specify the certificate file using the `TLS_CERTIFICATE` environment variable or the `--cert` CLI option.
+2. Specify the key file using the `TLS_PRIVATE_KEY` environment variable or `--key` CLI option.
 
-{{% note %}}
+    {{% note %}}
 If both the TLS certificate and key are in the same file, specify them using the `TLS_CERTIFICATE` environment variable (or the `--cert` CLI option).
-{{% /note %}}
+    {{% /note %}}
+
+3. _(Optional)_ Specify which TLS cipher suites to allow using the `TLS_CIPHERS` or `--tls-ciphers` CLI option.
```
Comment (jstirnaman) [inline content/chronograf/v1.9/administration/managing-security.md:564]:
```suggestion
3. _(Optional)_ To specify which TLS cipher suites to allow, use the `TLS_CIPHERS` environment variable or the `--tls-ciphers` CLI option.
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 8]
```diff
@@ -553,29 +554,56 @@ All Chronograf command line options have corresponding environment variables.
...
+    Chronograf supports all cipher suites in the
+    [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants)
+    and, by default, allows all of them.
+4. _(Optional)_ Specify the minimum and maximum TLS versions to allow with the
+    `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
+    `--tls-min-version` and `--tls-max-version` CLI options.
+    By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
+    unlimited.
```
Comment (jstirnaman) [inline content/chronograf/v1.9/administration/managing-security.md:572]:
```suggestion
4. _(Optional)_ To specify the minimum and maximum TLS versions to allow, use the
    `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
    `--tls-min-version` and `--tls-max-version` CLI options.
    By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
    unlimited.
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 9]
```diff
@@ -553,29 +554,56 @@ All Chronograf command line options have corresponding environment variables.
...
+  -e TLS_PRIVATE_KEY=/certs/my.key \
+  -e TLS_CIPHERS=TLS_RSA_WITH_AES_256_CBC_SHA,TLS_AES_128_GCM_SHA256 \
+  -e TLS_MIN_VERSION=tls1.2 \
+  -e TLS_MAX_VERSION=tls1.3 \
+  chronograf:{{< current-version >}}
 ```
 
-### Testing with self-signed certificates
+### Test with self-signed certificates
 In a production environment you should not use self-signed certificates, but for testing it is fast to create your own certificates.
```
Comment (jstirnaman) [inline content/chronograf/v1.9/administration/managing-security.md:607]:
```suggestion
To test your setup, you can use a self-signed certificate.

{{% warn %}}
Don't use self-signed certificates in production environments.
{{% /warn %}}

```

**Review by jstirnaman — APPROVED** [review]
Some minor suggestions.

**Pull request merged**